src/site/apt/index.apt.vm

```diff
@@ -26,14 +26,14 @@
 ${project.name}
 
   The Toolchains Plugins enables sharing configuration across plugins,
-  for example to make sure the plugins like compiler, surefire, javadoc, webstart etc. all use the same JDK for execution.
+  for example to make sure the plugins like compiler, surefire, javadoc, webstart etc. all use the same JDK for execution independantly from JRE used to run Maven itself.
```

Comment:
If there is place for adverb, it could be `independ*e*ntly` I think.

Reply:
fixed, thank you